examples/example_resources/lib/gen/assets.gen.dart

```diff
@@ -138,11 +142,14 @@ class SvgGenImage {
 
   final String _assetName;
 
+  static const String package = 'example_resources';
+
   SvgPicture svg({
     Key? key,
     bool matchTextDirection = false,
     AssetBundle? bundle,
-    String? package = 'example_resources',
+    @Deprecated('Do not use package for a package asset')
```

Comment:
In the deprecated user, what should the developer do instead?

"Package is deprecated as it now defaults to correct package" ?

Reply:
They should *not use package for a package asset*, which literally means not using it.

Reply:
Ah and it's `@deprecated` actually.